docs/content/guide/migration.ngdoc

```diff
@@ -30,6 +31,1116 @@ which drives many of these changes.
+    selected.
+
+
+
+- **input[type=radio]** now uses strict comparison (`===`) to determine its "checked" status.
+  ([Details](guide/migration#commit-5ac7da))
+
+- The improved support for **input[type=range]** means that the behaviour of range inputs (when
+  bound to `ngModel`) has changed. ([Details](guide/migration#commit-913016))
+
+- **ngTransclude** now treats whitespace-only transclusion content as empty and uses the fallback
+  content instead. ([Details](guide/migration#commit-32aa7e))
+
+- **ngAria/ngModel** no longer overrides the default `$inEmpty()` method for custom
+  `checkbox`-shaped controls. ([Details](guide/migration#commit-975a61))
+
+
+<br />
+Below is the full list of breaking changes:
+
+- Core:
+  - [Directives](guide/migration#migrate1.5to1.6-ng-directives)
+    - [form](guide/migration#migrate1.5to1.6-ng-directives-form)
+    - [input[radio]](guide/migration#migrate1.5to1.6-ng-directives-input[radio])
+    - [input[range]](guide/migration#migrate1.5to1.6-ng-directives-input[range])
+    - [ngBind](guide/migration#migrate1.5to1.6-ng-directives-ngBind)
+    - [ngModel](guide/migration#migrate1.5to1.6-ng-directives-ngModel)
+    - [ngModelOptions](guide/migration#migrate1.5to1.6-ng-directives-ngModelOptions)
+    - [ngTransclude](guide/migration#migrate1.5to1.6-ng-directives-ngTransclude)
+    - [select](guide/migration#migrate1.5to1.6-ng-directives-select)
+  - [Services](guide/migration#migrate1.5to1.6-ng-services)
+    - [$compile](guide/migration#migrate1.5to1.6-ng-services-$compile)
+    - [$http](guide/migration#migrate1.5to1.6-ng-services-$http)
+    - [$interpolate](guide/migration#migrate1.5to1.6-ng-services-$interpolate)
+    - [$location](guide/migration#migrate1.5to1.6-ng-services-$location)
+    - [$q](guide/migration#migrate1.5to1.6-ng-services-$q)
+  - [Miscellaneous](guide/migration#migrate1.5to1.6-ng-misc)
+    - [jqLite](guide/migration#migrate1.5to1.6-ng-misc-jqLite)
+    - [decorator()](guide/migration#migrate1.5to1.6-ng-misc-decorator)
+- Modules:
+  - [ngAria](guide/migration#migrate1.5to1.6-ngAria)
+    - [$aria](guide/migration#migrate1.5to1.6-ngAria-$aria)
+    - [ngClick](guide/migration#migrate1.5to1.6-ngAria-ngClick)
+    - [ngModel](guide/migration#migrate1.5to1.6-ngAria-ngModel)
+  - [ngMock](guide/migration#migrate1.5to1.6-ngMock)
+    - [$httpBackend](guide/migration#migrate1.5to1.6-ngMock-$httpBackend)
+  - [ngResource](guide/migration#migrate1.5to1.6-ngResource)
+    - [$resource](guide/migration#migrate1.5to1.6-ngResource-$resource)
+  - [ngRoute](guide/migration#migrate1.5to1.6-ngRoute)
+    - [$route](guide/migration#migrate1.5to1.6-ngRoute-$route)
+
+
+<br />
+<a name="migrate1.5to1.6-ng-directives"></a>
+### Core: _Directives_
+
+<a name="migrate1.5to1.6-ng-directives-form"></a>
+#### **form**:
+
+<minor />
+Due to [9e24e7](https://github.com/angular/angular.js/commit/9e24e774a558143b3478536911a3a4c1714564ba),
```

Comment:
Can we make all the "Due to XXXX, " bits bold? This will help to distinguish the start of a new BC.